### PR TITLE
glayout folder restructure

### PR DESCRIPTION
**Pull request by chetanyagoyal**
Cumulative PR for #307 and #304, all conflicts resolved

### CODE REVIEW

**Comment by chetanyagoyal** [comment]
@msaligane this can be merged

File: openfasoc/generators/glayout/glayout/flow/blocks/cross_coupled_inverter/CrossCoupledInverters_cell.py
Comment (alibillalhammoud) [inline openfasoc/generators/glayout/glayout/flow/blocks/cross_coupled_inverter/CrossCoupledInverters_cell.py:1]: why is the cell in this file twice?

File: openfasoc/generators/glayout/glayout/syntaxer/relational.py [hunk 1]
```diff
@@ -377,7 +377,7 @@ def test(cls):
         tests.append([["via","via stack","via_stack"],"via_stack","glayout.flow.primitives.via_gen"])
         tests.append([["via array","via_array"],"via_array","glayout.flow.primitives.via_gen"])
         tests.append([["nlp"],"test_stream","../interpreter/deprecated/../deprecated/practice_stream.py"])
-        tests.append([["nmos","nfet"],"nmos","./glayout/primitives/fet.py"])
+        tests.append([["nmos","nfet"],"nmos","./glayout/flow/primitives/fet/fet.py"])
```
Comment (alibillalhammoud) [inline openfasoc/generators/glayout/glayout/syntaxer/relational.py:380]: fix

**Review by alibillalhammoud — CHANGES_REQUESTED** [review]
one issue with cross coupled inverters. Looks like the file has 2 compilations?

**Comment by chetanyagoyal** [comment]
resolved all issues

**Review by msaligane — APPROVED** [review]

**Pull request merged**